CMakeLists.txt

```diff
@@ -87,13 +74,27 @@ if(APPLE)
     endif()
     message(STATUS "Using MoltenVK repo location at ${MOLTENVK_REPO_ROOT}")
 endif()
-message(STATUS "Using find_package to locate Vulkan")
-find_package(Vulkan)
+
 find_package(VulkanHeaders REQUIRED CONFIG)
-get_filename_component(Vulkan_LIBRARY_DIR ${Vulkan_LIBRARY} DIRECTORY)
-message(STATUS "Vulkan FOUND = ${Vulkan_FOUND}")
-message(STATUS "Vulkan Lib Dir = ${Vulkan_LIBRARY_DIR}")
-message(STATUS "Vulkan Lib = ${Vulkan_LIBRARY}")
+
+# Find and create the Vulkan::Vulkan library using the VULKAN_LOADER_INSTALL_DIR for the vulkan loader and VULKAN_HEADERS_INSTALL_DIR for the include
```

Comment:
If you add VULKAN_LOADER_INSTALL_DIR to CMAKE_PREFIX_PATH the code becomes simlper. And works better for system package managers.

```
    find_library(Vulkan_LIBRARY NAMES vulkan-1 vulkan)
```

Note: Double check if the PATH_SUFFIXES part is needed for windows.